### PR TITLE
Update price with asterisk and vertical align

### PR DESCRIPTION
**Pull request by IvanPresmytsky**
# What does this PR do:

   * Extend Price component to support vertical align customization for its items by line-height variables
      Fixed issue #187 

   * Extend Price component to support ability to add symbols after the price (asterisk)
      Fixed issue #194 

   * Modify horizontal margins between currency, integers and decimals.

   * Modify underline height for small and extra small states

# Where should the reviewer start:

    Diffs

Cheaptickets

extendend-price-component cheaptickets

Budgetair

extending-price-component budgetair -after

### CODE REVIEW

File: components/price/price.md [hunk 1]
```diff
@@ -22,9 +29,9 @@ Price with discount:
 Sizes:
 
     <div>
-      <Price size="xs" value={50153.30} /><br/><br/>
-      <Price size="s" value={50153.30} /><br/><br/>
-      <Price size="m" underlined value={50153.30} /><br/><br/>
-      <Price size="l" underlined value={50153.30} /><br/><br/>
-      <Price size="xl" underlined value={50153.30} /><br/><br/>
+      <Price showAsterisk size="xs" underlined value={50153.30} /><br/><br/>
+      <Price showAsterisk size="s" underlined value={50153.30} /><br/><br/>
```
Comment (iwwwi) [inline components/price/price.md]: I would suggest to leave few examples without asterisk or add both cases of sizes (with and without asterisk)

**Comment by yurist38** [comment]
@IvanPresmytsky Thank you for reacting on my message and updating description with screenshot! Maybe I don't understand something but I don't see any asterisk on the it... Is that an old version on the screenshot?

**Comment by IvanPresmytsky** [comment]
@yurist38 
Thanks for remarks.
Updated

**Comment by yurist38** [comment]
Thanks @IvanPresmytsky ! But it looks like space is missing between `€` sign and price number, isn't it?

**Comment by IvanPresmytsky** [comment]
@yurist38 
Updated :)

File: components/price/price.js [hunk 2]
```diff
@@ -100,6 +101,15 @@ function Price(props) {
       )
     : null;
 
+  const asteriskSymbol = '*';
```
Comment (froskie) [inline components/price/price.js]: What's the purpose of creating this?

Comment (IvanPresmytsky) [reply components/price/price.js]: @froskie 
May be in the future we will need another symbol/symbols. :-)
But for now this variable is redundant. I agree.
Should I remove it?

**Pull request merged**